warehouse/models/staging/transit_database/stg_transit_database__eligibility_programs.sql

```diff
@@ -0,0 +1,31 @@
+{{ config(materialized='table') }}
+
+WITH
+latest AS (
+    {{ get_latest_external_data(
+        external_table = source('airtable', 'california_transit__eligibility_programs'),
+        order_by = 'dt DESC, time DESC'
+        ) }}
+),
+
+stg_transit_database__eligibility_programs AS (
+    SELECT
+        eligibility_program_id AS key,
+        {{ trim_make_empty_string_null(column_name = "program") }},
+        unnested_administering_entity AS administering_entity,
+        unnested_eligibility_types AS eligibility_types,
```

Comment:
same comment as above -- this would become administering_entity_organization_key and eligibility_type_rider_requirement_key and services below would become service_key